_posts/2023-03-02-q1-survey-results.md

```diff
@@ -0,0 +1,46 @@
+
+---
+
+Thank you to the OpenSearch community members who responded to our 2023-Q1 quarterly survey! Our first quarterly survey focused on understanding who our users are, what they use OpenSearch for, and establishing a baseline for how they feel about the current user experience on OpenSearch. We are thrilled to report a NPS (net promotor score) of **57.8** and a UX health score of **77.2** with **97** community members taking part in the survey. 
+
+We’ve recently launched a research program to get a holistic view of of our users’ experience. The program is designed to evaluate a user’s journey comprehensively, and amongst various research methodologies, includes a quarterly survey. In addition to quarterly surveys, the team also utilizes deep-dives, which are qualitative research studies that include conversations with users to understand their use cases, requirements, and OpenSearch experience. The surveys, going live once a quarter, will ask our community members to provide feedback on new features, existing experiences on OpenSearch, and more. This will help the OpenSearch team prioritize improvements to best serve the community. For the deep-dives, we invite community members who are willing to help us understand their use case, to talk to members of the OpenSearch team. After our Q2-2023 survey is live in March, you can access it from the landing page of [OpenSearch.org](http://opensearch.org/):
+
+![OpenSearch.org website](../assets/media/blog-images/2023-03-02-q1-survey-results/q1-survey-results.png)
+
+[Image: survey.png]
+
+Here are some key insights from the Q1 survey:
+
+* We obtained our first NPS and UX health score, and will use these as a baseline to compare against results in future quarters.
+* Previously, we were unsure about the split of *search* and *analytics* use cases and were curious if there was an overlap in the way users were utilizing OpenSearch. Around 1/3 (one-third) of the sample that took the survey in Q1 indicated that they used OpenSearch as *both a search and analytics solution*. 
+* Users identified with four roles in OpenSearch: 
+  - _Infrastructure Set Up_ - responsible for deploying OpenSearch, for allocation of resources, responsible for allocation of initial resources, and any action that impacts the billing and usage or resources
+  - _OpenSearch Admin_ - responsible for asset and user management on OpenSearch
+  - _Dashboard Creator or Data Producer_ - responsible for preparing data or dashboards for their own use or end user consumption
+  - _Dashboard or Data Consumer_ - responsible for consuming data or dashboards
+  
+* While users do not use the above labels to self-identify, the motivations described in the survey for each of these roles resonated with them.
+* 48.8%, or almost half of the community members who took the survey, self-identified themselves to be in an *administrative* role and almost half of the sample associated with all four roles.
```

Comment:
```suggestion
* 48.8%, or almost half of the community members who took the survey, self-identified as being in an *administrative* role, and almost half of the sample self-identified with all 4 roles.
```

Reply:
Is "or almost half of the community members who took the survey" redundant, given that we provide the percentage?

Reply:
Not sure. Either there is overlap or no one self-identified as being as being any of the other roles exclusively.

Reply:
I just meant that the "or almost" phrase essentially says what "48.8%" says, but there's value in including it for emphasis, so let's leave as is.